README.md

```diff
@@ -19,7 +19,7 @@ This client supports the following Google Cloud Platform services at a [Beta](#v
 -  [Google Cloud Datastore](#google-cloud-datastore-beta) (Beta)
 -  [Google Cloud Storage](#google-cloud-storage-beta) (Beta)
 -  [Cloud Spanner](#cloud-spanner-beta) (Beta)
--  [Google Cloud Translate](#google-translate-beta) (Beta)
+-  [Google Cloud Translation](#google-translate-beta) (Beta)
```

Comment:
This will be a broken link. In general I recommend clicking the "View" button for README file changes - github will show you the rendered view, and you can check links.